vtm/src/org/oscim/tiling/source/mapfile/MapDatabase.java

```diff
@@ -872,11 +868,10 @@ private boolean processWays(QueryParameters queryParameters, ITileDataSink mapDa
             byte numberOfTags = (byte) (specialByte & WAY_NUMBER_OF_TAGS_BITMASK);
 
             if (numberOfTags != 0) {
-
+                // Reset to ensure experimental tileBitmask tags aren't included
+                e.tags.clear();
```

Comment:
Probably not needed, since `ReadBuffer.readTags` clears tags at start anyway?

Reply:
Sounds good!